# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/http/Http3Session.h
# Commit: 23b864e14db0
# Full Hash: 23b864e14db0cf874d0e126b6299c8c44922dde3
# Author: Dana Keeler <dkeeler@mozilla.com>
# Date: 2022-11-16 09:31:06
# Regressor Bug: 1791633
# File Overlap Count: 1
# Description:
#   Bug 1791633 - separate nsITLSSocketControl from nsITransportSecurityInfo r=necko-reviewers,kershaw,jschanck
#   
#   Depends on D160311
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D160313
# ==============================================================================

diff -r 0bcba3375ec0 -r 23b864e14db0 netwerk/protocol/http/Http3Session.h
--- a/netwerk/protocol/http/Http3Session.h	Tue Nov 15 20:03:29 2022 +0000
+++ b/netwerk/protocol/http/Http3Session.h	Tue Nov 15 20:03:29 2022 +0000
@@ -7,17 +7,18 @@
 #ifndef Http3Session_H__
 #define Http3Session_H__
 
+#include "HttpTrafficAnalyzer.h"
+#include "mozilla/UniquePtr.h"
+#include "mozilla/WeakPtr.h"
+#include "mozilla/net/NeqoHttp3Conn.h"
+#include "nsAHttpConnection.h"
+#include "nsDeque.h"
 #include "nsISupportsImpl.h"
 #include "nsITimer.h"
 #include "nsIUDPSocket.h"
-#include "mozilla/net/NeqoHttp3Conn.h"
-#include "nsAHttpConnection.h"
 #include "nsRefPtrHashtable.h"
+#include "nsTHashMap.h"
 #include "nsWeakReference.h"
-#include "HttpTrafficAnalyzer.h"
-#include "mozilla/UniquePtr.h"
-#include "mozilla/WeakPtr.h"
-#include "nsDeque.h"
 
 /*
  * WebTransport